src/transformers/generation/candidate_generator.py

```diff
@@ -157,6 +157,13 @@ def __init__(
         self.generation_config.return_dict_in_generate = True
         self.generation_config.output_scores = True
 
+        # avoid unnecessary warnings that min_length is larger than max_new_tokens
+        input_length = input_ids.shape[-1]
+        min_new_tokens = self.generation_config.min_new_tokens if self.generation_config.min_new_tokens else 0
```

Comment:
Is this checking for None-ness? Or can it be `False`? Otherwise, defaulting to 0 if it's 0 is a superfluous check

Reply:
For Noneness, I'll specify it as `not None`